ocs_ci/ocs/constants.py

```diff
@@ -1434,6 +1434,18 @@
         "odf-multicluster-orchestrator",
         "odf-operator",
     ],
+    "4.13": [
```

Comment:
I see no change from 4.12. Thinking about if we can change it to prevent the duplications.

Maybe just something like:
DISCON_CL_REQUIRED_PACKAGES_PER_ODF_VERSION["4.13"] = DISCON_CL_REQUIRED_PACKAGES_PER_ODF_VERSION["4.12"]
?

Reply:
Nice idea, updated.